__tests__/app.test.js

```diff
@@ -66,6 +66,7 @@ describe('GET', () => {
               title: expect.any(String),
               topic: expect.any(String),
               votes: expect.any(Number),
+              comment_count: expect.any(String)
```

Comment:
As we're requesting something specific (an article by its id) we can assert for specific values here. Still utilising `expect.objectContaining` as you are, but we should be able to determine exactly the values here, including how many comments article 1 has - you can keep the assertion for `created_at` as is 👍